tools/report-converter/tests/unit/spotbugs_output_test_files/files/Assign.java

```diff
@@ -0,0 +1,6 @@
+public class Assign {
```

Comment:
Could you add a simple Makefile here, with a build target to execute SpotBugs on this file to produce the expected assign.xml? What to see what options were enabled or disabled.
Maybe it is documented somewhere I've just missed it.